### ISSUE TITLE
Form field- inputs should be associated with headlines 

### ISSUE THREAD

**Issue by ryanhofdotgov** [18F/e-QIP-prototype#193]
From e-QIP-prototype-truetandem created by [Jay-Neighbours-I72](https://github.com/Jay-Neighbours-I72) : ryanhofdotgov/e-QIP-prototype-truetandem#2666  

Form wide - inputs should be associated with headlines (for context)

Examples:
- Spouse name - screen reader reads "first name" - need to add context to it is clear that spouse first name is being asked for
- "Initial select box should be assicaited with the form it applies to,
- Suffix selection should be associated with ""Suffix"",
- All buttons need to be associated with instructions, ex ""Mothers maiden name? yes no"" Fieldset and legend would work
- Phone input fields need proper labels for screen reader
- Date of birth fields should indicate they are for a date of birth"
